prow/plugins.yaml

```diff
@@ -72,112 +69,64 @@ approve:
   implicit_self_approve: true
 
 plugins:
-  istio/istio:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
-  istio/api:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
-  istio/installer:
-  - trigger
+  istio:
+    - size
+    - trigger
+    - wip
 
-  istio/operator:
-  - assign
-  - hold
-  - slackevents
-  - trigger
-  - wip
+  istio/istio:
+    - approve
+    - assign
+    - blunderbuss
+    - golint
+    - lgtm
+    - lifecycle
+    - verify-owners
 
   istio/proxy:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
-  sebastienvas/core:
-  - assign
-  - hold
-  - lgtm
-  - lifecycle
-  - trigger
+    - approve
+    - assign
+    - blunderbuss
+    - golint
+    - lgtm
+    - lifecycle
+    - verify-owners
 
   istio/test-infra:
-  - approve
```

Comment:
Will you please leave the indentation at its previous level? This should reduce the deltas and make it easier to review.